docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.6"
+
+services:
+  build:
+    build:
+      context: .
+    environment:
+      - API_URL=${API_URL}
+    volumes:
+      - ./dist:/dist
```

Comment:
For more clarity as to the use cases of volumes: https://documentation.codeship.com/pro/builds-and-configuration/docker-volumes/. It lets other containers access the volumes and in CICD it often helps us to share data/files across steps that may happen in other containers